shell/platform/common/cpp/client_wrapper/method_result_functions_unittests.cc

```diff
@@ -2,11 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/method_result_functions.h"
-
 #include <functional>
 #include <string>
 
+#include "flutter/shell/platform/common/cpp/client_wrapper/include/flutter/method_result_functions.h"
```

Comment:
I thought that the header corresponding to the cc file always came first... Am I just wrong? Is it supposed to come after the system headers like this? Or is that only in the non-test files?

Reply:
It should; the style guide is explicit about this. But for reasons that aren't clear to me, clang-format in the engine repo keeps moving the header down in test files.

I keep meaning to look into it, then forgetting. I'll file a bug. Maybe it only understands `_test`, and not `_unittests`? But I swear it used to leave it alone.